cms/static/cms/js/modules/cms.modal.js

```diff
@@ -366,29 +469,50 @@
                 });
             },
 
-            _endMove: function () {
+            /**
+             * Initiates the stop move event from `_startResize`.
+             *
+             * @method _stopMove
+             * @private
+             */
+            _stopMove: function _stopMove() {
                 this.ui.shim.hide();
-
-                this.ui.body.off('pointermove.cms').removeAttr('data-touch-action');
+                this.ui.body
+                    .off(this.pointerMove)
+                    .off(this.pointerUp)
+                    .removeAttr('data-touch-action');
             },
 
-            _startResize: function (initial) {
+            /**
+             * Initiates the start resize event from `_events`.
+             *
+             * @method _startResize
+             * @param originEvent {Object} passes starting event
```

Comment:
maybe it's worth noting that this event is coming from pointer events polyfill, that's why we are using `originalEvent` property on the event object, but i think naming the param `originEvent` is misleading